lib/line/line.ex

```diff
@@ -18,5 +18,7 @@ defmodule Line do
           }
         ]
       })
+
+    :ok
```

Comment:
or we could return nothing/nil? Not sure what is idiomatic elixir for side effects